docs/administration-guide/03-metadata-editing.md

```diff
@@ -161,6 +160,21 @@ You can also designate a column as the table's **primary key** or **foreign key*
 - **Other**
   - Field containing JSON
 
+### Casting to a specific data type
+
+You can manually cast text and number columns to date fields. Casting is useful if your date/timestamps are stored as text or number fields in your database, but you want Metabase to treat them like date values (so it'll do things like present calendar pickers for filters). Casting is different from changing the field type in Metabase, which only modifies semantic information to give people more context about that field, like setting a date field as a "Cancellation date". By contrast, casting changes the underlying type, so that, for example, Metabase will interpret a string field as if it were a date.
```

Comment:
```suggestion
You can manually cast text and number columns to date fields. Casting is useful if your date/timestamps are stored as text or number fields in your database, but you want Metabase to treat them like date values (so it'll do things like present calendar pickers for filters). Casting is different from changing the field type in Metabase, which only modifies semantic information to give people more context about that field, like setting a date field as a "Cancellation date." By contrast, casting changes the underlying type so that, for example, Metabase will interpret a string field as if it were a date.
```